### PR TITLE
Add support for next episodes + Endpoint refactoring

### PR DESCRIPTION
**Pull request by LucasCairns**
- Update tests to match new endpoints
- Reduce complexity of test suite
- Reduce complexity of api responses

### CODE REVIEW

File: moj-be/modules/custom/moj_resources/src/ContentApiClass.php [hunk 1]
```diff
@@ -5,92 +5,194 @@
...
-    }
-    /**
-     * API resource function
-     *
-     * @param [string] $lang
-     * @return array
-     */
-    public function ContentApiEndpoint($lang, $nid)
-    {   
-        $this->lang = $lang;
```
Comment (romidane) [inline moj-be/modules/custom/moj_resources/src/ContentApiClass.php]: We need to re-add this line.. `translateNode` uses it..
I'd prefer `translateNode` to take it as an arg but let's be consistent :)

File: moj-be/modules/custom/moj_resources/src/SeriesContentApiClass.php [hunk 2]
```diff
@@ -68,24 +68,122 @@ public function __construct(
...
+          return $key;
+        }
+      }
+    }
+
+    $episode_index = indexOf(function($value) use ($episode_id) {
+      return $value['episode_id'] == $episode_id;
+    }, $series);
+
+    // $episode_index = array_reduce($series, function($acc, $curr) use ($episode_id) {
```
Comment (LucasCairns) [inline moj-be/modules/custom/moj_resources/src/SeriesContentApiClass.php]: Remove

File: moj-node/server/repositories/hubFeaturedContent.js [hunk 3]
```diff
@@ -1,107 +1,100 @@
...
 
   function music({ establishmentId }) {
-    return hubContentFor({
-      query: {
-        _number: 4,
-        _category: 785,
-        _prison: establishmentId,
-      },
+    return contentFor({
+      _number: 4,
```
Comment (romidane) [inline moj-node/server/repositories/hubFeaturedContent.js]: `number` not `_number`

File: moj-node/server/services/hubContent.js [hunk 4]
```diff
@@ -28,7 +28,9 @@ module.exports = function createHubContentService({
 
   async function media(establishmentId, data) {
     const id = prop('id', data);
+
```
Comment (romidane) [inline moj-node/server/services/hubContent.js]: extra space?

File: moj-node/server/repositories/hubFeaturedContent.js (hunk 3 above)
Comment (LucasCairns) [inline moj-node/server/repositories/hubFeaturedContent.js]: Remove _

**Pull request merged**